can_tools/utils.py

```diff
@@ -46,17 +46,25 @@ def load_most_recent_cdc(
     pass
 
 
-def find_unknown_variable_id(df: pd.DataFrame, engine: Engine = None):
```

Comment:
update the return types here so these functions can be used in `_create_put_error_msg` below